### ISSUE TITLE
Too many options causes jquery to crash

### ISSUE THREAD

**Issue by beck24** [hypeJunction/elgg_tokeninput#4]
One field that has 18 options selected causes jquery to crash when the form is loaded.  It handles it on input, but crashes when trying to render the field when the options have been saved.

**Comment by beck24** [comment]
ok, it's not the number of options - it's options that use html encoded things that break after being json encoded

eg.
```Hello &quot;world&quot;```

becomes:
```{"test":"Hello "world""}```

Need to double-encode htmlentities when exporting entities I think

**Comment by hypeJunction** [comment]
Are you using the latest version? I thought I have fixed this along the way

**Comment by beck24** [comment]
oh maybe not, it's on EN so it could be an older version.  I'll check

**Comment by beck24** [comment]
Manifest says 1.0.2.0 which matches the current in the repo for 1.8 compatibility.  Did you fix it for 1.9+?

**Comment by beck24** [comment]
Yep, I see the fix in the 1.9+ version, it's not present it the 1.8 version

**Comment by hypeJunction** [comment]
Yeah, sorry. I gave up on 1.8. Too much work maintaining all of them

**Comment by beck24** [comment]
No prob, I'll submit a PR or something when I get time.  I'm using this in a bunch of 1.8 places.
